package.json

```diff
@@ -6,6 +6,9 @@
     "email": "info@acorn.io",
     "url": "https://acorn.io"
   },
+  "files": [
+    "bin"
+  ],
```

Comment:
I don't think we need this since we're already including the `bin` directory with the `extraResources` field on the config we're passing to electron builder.